lock/client/client.go

```diff
@@ -85,8 +85,6 @@ func (d *Driver) reportMountEndpoint(endpoint string, ut *config.UseMount) error
 		return err
 	}
 
-	fmt.Printf("%#v\n", d)
```

Comment:
It's a removal; a spurious printf. :)

On 8 Mar 2016, at 15:48, Madhav Puri wrote:

>> @@ -85,8 +85,6 @@ func (d *Driver) reportMountEndpoint(endpoint 
>> string, ut *config.UseMount) error
>> 		return err
>> 	}
>>
>> -	fmt.Printf("%#v\n", d)
>
> looks like this is a left over print or may be this should be 
> `log.Debugf()` ?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/171/files#r55449906